downstream/modules/automation-hub/con-network-storage-reqs.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// assembly-deploying-high-availability-hub.adoc
+
+
+[id="con-network-storage-reqs"]
+
+= Network Storage Installation Requirements
+
+If you intend to install a HA {HubName} using a network storage on the {HubName} nodes itself, you must first install and use _firewalld_ to open the necessary ports as required by your shared storage system before running the {PlatformNameShort} installer.
```

Comment:
firewalld markup should be in monospace since its a service. IBM suggests bold monospace for commands which I think is fine in this guide.